octobot_trading/personal_data/positions/position.py

```diff
@@ -320,6 +320,17 @@ def _is_update_increasing_size(self, size_update):
             return size_update > constants.ZERO
         return size_update < constants.ZERO
 
+    def _is_update_decreasing_size(self, size_update):
```

Comment:
Why not using `not  _is_update_increasing_size()` ?

Reply:
`_is_update_increasing_size` is a strict comparison that returns True on a 0 size update, in this particular case, we don't want to update realized pnl and create a 0 pnl value transaction

Reply:
It's also an issue when `size_update` value is 0 as `_is_update_increasing_size` returns False while it's not actually decreasing the position

Reply:
But `size_update = 0` isn't actually increasing the position size either

Reply:
Right, and in `_update_size` we need to know if the position is really decreasing of we will create an empty transaction, which makes no sense in y opinion

Reply:
I understand your fix but I don't understand why you didn't just replace `_is_update_increasing_size` by `_is_update_decreasing_size`?

Reply:
We need both `_is_update_increasing_size` as  `_is_update_increasing_size` is also used elsewhere

Reply:
`_update_size` is the only method that use  `_is_update_increasing_size`

Reply:
Right, to me it makes sense to keep it as it's not an obvious computation and therefore having it in a dedicated method is good. Maybe you have a different idea ?

Reply:
I was just wondering if `_is_update_increasing_size` should be kept because its only use is `_update_size` and we already have the information with `not` `_is_update_decreasing_size`'s result.

Reply:
> I was just wondering if `_is_update_increasing_size` should be kept because its only use is `_update_size` and we already have the information with `not` `_is_update_decreasing_size`'s result.

Not really because with an  `_update_size` of 0, `_is_update_increasing_size` and `_is_update_decreasing_size` should both be false, and therefore `not _is_update_decreasing_size` is not equal to `_is_update_increasing_size`

Reply:
Ok I got it :+1: